windows-rdp/main.test.ts

```diff
@@ -0,0 +1,130 @@
+import { describe, expect, it } from "bun:test";
+import {
+  TerraformState,
+  runTerraformApply,
+  runTerraformInit,
+  testRequiredVariables,
+} from "../test";
+
+type TestVariables = Readonly<{
+  agent_id: string;
+  resource_id: string;
+  admin_username?: string;
+  admin_password?: string;
+}>;
+
+function findWindowsRpdScript(state: TerraformState): string | null {
```

Comment:
nit: typo: should be `findWindowsRdpScript`
alternatively: `findWindowsRDPScript`